app/renderer/components/ServerMonitor/ServerMonitor.js

```diff
@@ -235,20 +237,52 @@ export default class ServerMonitor extends Component {
     let logLineSection = logLines.slice(logLines.length - MAX_LOGS_RENDERED).map((line, i) => {
       let icn = leveler(line.level);
 
-      return (
+      let lineHtml = convert.toHtml(line.msg);
+      // using colors defined in: https://terminal.sexy/
+      if (lineHtml.includes('[Appium]') || lineHtml.includes('[Instrumentation]')) {
```

Comment:
i kinda don't want appium desktop to know about these specific log prefixes. one other idea would be to simply look at each line, and if it contains a prefix which hasn't been seen before, assign a new random color (or pick from a ring of colors)

Reply:
That is a good idea. I will do that.

The `inspector` will still be supported right?

Reply:
absolutely the inspector will be supported! it's all that appium desktop was really about in the first place

Reply:
@jlipps Makes sense I did always find it odd that in order to use the Appium Inspector in past, I would have to start an Appium server but most of the time I already had one running (one I configured by me and use for automation). I would always start the 2nd on a different port. Makes sense they are unbound from each other.

Additionally, I am not sure at the current time I see the time available to champion Appium Desktop into 2.0.

BUUUUUUT, at some point, it may be something helpful for newer people to Appium as now drivers are downloaded individually and it would be a way to have people visualize configuration options, as there has been inconsistency with required capabilities, prefix, and even data types for capabilities in documentation between Selenium and Appium in the past, they may have been resolved but maybe a good idea for an "Initialization Wizard" in a GUI form.